src/renderer/components/ui/FailedUsernameRegister.tsx

```diff
@@ -0,0 +1,97 @@
+import React from 'react'
+
+import Grid from '@material-ui/core/Grid'
+import Typography from '@material-ui/core/Typography'
+import { makeStyles, MuiThemeProvider } from '@material-ui/core/styles'
+
+import red from '@material-ui/core/colors/red'
+
+import Icon from './Icon'
+import Modal from './Modal'
+import LoadingButton from './LoadingButton'
+
+import exclamationMark from '../../static/images/exclamationMark.svg'
+
+import theme from '../../theme'
+
+const useStyles = makeStyles(() => ({
+  root: {
+    padding: theme.spacing(4)
+  },
+  icon: {
+    fontSize: '10rem',
+    color: red[500],
+    width: 80,
+    height: 70
+  },
+  message: {
+    wordBreak: 'break-all',
+    marginTop: 36,
+    fontWeight: 500
+  },
+  info: {
+    textAlign: 'center'
+  },
+  button: {
+    textTransform: 'none',
+    width: 187,
+    height: 60,
+    fontSize: 16,
+    lineHeight: '19px',
+    fontWeight: 'normal',
+    marginTop: 24
+  }
+}))
+
+interface FailedUsernameRegisterProps {
+  open: boolean
+  handleClose: () => void
+  openModalCreateUsername: () => void
+}
+
+export const FailedUsernameRegister: React.FC<FailedUsernameRegisterProps> = ({
+  open = false,
+  handleClose,
+  openModalCreateUsername
+}) => {
+  const classes = useStyles({})
+
+  return (
+    <MuiThemeProvider theme={theme}>
```

Comment:
you shouldn't be introducing this provider in every component

In real application it's provided somewhere up in React DOM tree.
You should provide it in tests